pkg/flavor/kubernetes/flavor.go

```diff
@@ -281,6 +293,14 @@ func getWwpnsFromNode(node *model.Node) []string {
 	return wwpns
 }
 
+func getChapUserFromEnvironment() string {
+	return os.Getenv("CHAP_USER")
```

Comment:
Can we use here, chapUserEnvKey const?

Reply:
Importing constants creates a cyclic import dependency.

Reply:
Can we update https://github.com/hpe-storage/csi-driver/blob/master/pkg/driver/node_server.go#L2059
https://github.com/hpe-storage/csi-driver/blob/master/pkg/driver/node_server.go#L2060 to call the new method?